checkstyle.json

```diff
@@ -517,6 +517,22 @@
 		},
 		{
 			"props": {
+				"severity": "INFO",
+				"ignorePackages": [
```

Comment:
Yeah, if you wanted to do this correctly, you'd have to rebuild parts of the Haxe compiler, resolving haxelibs etc... That seems unreasonable.